ui/src/authProvider.js

```diff
@@ -5,6 +5,20 @@ import { baseUrl } from './utils'
 import config from './config'
 import { startEventStream, stopEventStream } from './eventStream'
 
+if (
+  config.auth &&
+  /^[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12}$/.test(config.auth.id)
```

Comment:
The problem is that `jwtDecode` would throw exception which may break module load. I'll keep this `if` condition for pre-check that we actually have auth to check, and I'll add `jwtDecode` inside of the `if` block.

Reply:
You could do something like:
```js
if (config.auth) {
  try {
    jwtDecode(config.auth.token)
    localStorage.setItem('token', config.auth.token)
    localStorage.setItem('userId', config.auth.id)
    ...
  } catch(e) {
    console.log(e)
  }
}
```

This would not break the module load, right?